Modules/Backup-VCSA/Backup-VCSA.psm1

```diff
@@ -1,204 +1,462 @@
+			$Connection = Connect-CisServer $global:DefaultVIServer 
+		} else {
+			$Connection = $global:DefaultCisServers
+		}
+	}
+	Process{
+		$BackupAPI = Get-CisService com.vmware.appliance.recovery.backup.job
+		try {
+			if ($ShowNewest) {
+				$results = $BackupAPI.list()
+				$results[0]
+			} else {
+				$BackupAPI.list()
+			}
+		} catch {
+			Write-Error $Error[0].exception.Message
+		}
+	}
+	End {}
 }
 
 Function Get-VCSABackupStatus {
 <#
-    .NOTES
-    ===========================================================================
+	.NOTES
+	===========================================================================
 	 Created by:   	Brian Graf
-     Date:          October 30, 2016
+	 Date:          October 30, 2016
 	 Organization: 	VMware
-     Blog:          www.vtagion.com
-     Twitter:       @vBrianGraf
+	 Blog:          www.vtagion.com
+	 Twitter:       @vBrianGraf
 	===========================================================================
 
 	.SYNOPSIS
 		Returns the ID, Progress, and State of a VCSA backup (vSphere 6.5 and higher)
-	
 	.DESCRIPTION
 		Returns the ID, Progress, and State of a VCSA backup
-
-	.EXAMPLE
+ 	.EXAMPLE
 		PS C:\> $backups = Get-VCSABackupJobs
-                $backups[0] | Get-VCSABackupStatus
-	
+				$backups[0] | Get-VCSABackupStatus
 	.NOTES
 		The BackupID can be piped in from the Get-VCSABackupJobs function and can return multiple job statuses
 #>
-    Param (
-        [parameter(ValueFromPipeline=$True)]
-        [string[]]$BackupID
-    )
- Begin {
-        if (!($global:DefaultCisServers)){ 
-            [System.Windows.Forms.MessageBox]::Show("It appears you have not created a connection to the CisServer. You will now be prompted to enter your vCenter credentials to continue" , "Connect to CisServer") | out-null
-            $Connection = Connect-CisServer $global:DefaultVIServer 
-        } else {
-            $Connection = $global:DefaultCisServers
-        }
-        
-        $BackupAPI = Get-CisService com.vmware.appliance.recovery.backup.job
- }
-    Process{
-       
-        foreach ($id in $BackupID) {
-            $BackupAPI.get("$id") | select id, progress, state
-        }
-        
+	Param (
+		[parameter(ValueFromPipeline=$True)]
+		[string[]]$BackupID
+	)
+	Begin {
+		if (!($global:DefaultCisServers)){ 
+			[System.Windows.Forms.MessageBox]::Show("It appears you have not created a connection to the CisServer. You will now be prompted to enter your vCenter credentials to continue" , "Connect to CisServer") | out-null
+			$Connection = Connect-CisServer $global:DefaultVIServer 
+		} else {
+			$Connection = $global:DefaultCisServers
+		}
+		$BackupAPI = Get-CisService com.vmware.appliance.recovery.backup.job
+	}
+	Process{
+		foreach ($id in $BackupID) {
+			$BackupAPI.get("$id") | select id, progress, state
+		}
+	}
+	End {}
+}
+
+Function Create-VCSASchedule {
```

Comment:
To follow PowerShell standards, verb should be one of the approved ones, for create operation it is 'New'. Rename to New-VCSASchedule